apps/client/src/register/components/registerForm.tsx

```diff
@@ -0,0 +1,143 @@
+import { Button } from '@mui/material';
+import FormInput from '../../components/FormInput';
+import React, { useState } from 'react';
+import axios from 'axios';
+
+const RegisterForm: React.FC = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [passwordError, setPasswordError] = useState('');
+  const [confirmPasswordError, setConfirmPasswordError] = useState('');
+
+  const validatePassword = (password: string) => {
+    if (password.length < 6) {
+      setPasswordError('Password must be at least 6 characters long');
+      return false;
+    } else {
+      setPasswordError('');
+      return true;
+    }
+  };
+
+  const handlePasswordChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const newPassword = e.target.value;
+    setPassword(newPassword);
+    validatePassword(newPassword);
+    // Check if confirm password matches the new password
+    if (confirmPassword && confirmPassword !== newPassword) {
+      setConfirmPasswordError('Passwords do not match');
+    } else {
+      setConfirmPasswordError('');
+    }
+  };
+
+  const handleConfirmPasswordChange = (
+    e: React.ChangeEvent<HTMLInputElement>
+  ) => {
+    const newConfirmPassword = e.target.value;
+    setConfirmPassword(newConfirmPassword);
+    if (password !== newConfirmPassword) {
+      setConfirmPasswordError('Passwords do not match');
+    } else {
+      setConfirmPasswordError('');
+    }
+  };
+
+  const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    if (!validatePassword(password)) {
+      return;
+    }
+    if (password !== confirmPassword) {
+      setConfirmPasswordError('Passwords do not match');
+      return;
+    }
+    try {
+      const response = await axios.post('http://localhost:3000/register', {
+        email,
+        password,
+        firstName,
+        lastName,
+      });
+      console.log(response);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  return (
+    <div className="flex justify-center h-screen pt-14 bg-neutral-100">
+      <div className="flex flex-col items-center justify-center border bg-white rounded  w-96 h-[650px] gap-5 py-8">
+        <div>
+          <h1 className="text-2xl text-gray-700">Create new Account</h1>
+        </div>
+        <form className="flex flex-col gap-8" onSubmit={handleSubmit}>
+          <div className="flex flex-col gap-5">
+            <FormInput
```

Comment:
If you have MUI installed, please use it, don't mix styling systems. 
Here you can use [Stack](https://mui.com/material-ui/react-stack/) instead of div and [Typography](https://mui.com/material-ui/react-typography/) instead of h1